<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1512</width>
    <height>905</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>EhDbViewer</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <widget class="QLineEdit" name="txtSearchBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableSearchResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>871</width>
      <height>531</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>60</y>
      <width>152</width>
      <height>144</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btnTestListFullDb">
       <property name="text">
        <string>列出数据库</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenSelectedFolder">
       <property name="text">
        <string>打开选中目录</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTestEhRequest">
       <property name="text">
        <string>Eh请求</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTestProgressDiag">
       <property name="text">
        <string>Test Progress Dialog</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>60</y>
      <width>411</width>
      <height>583</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Image Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollImagePreview">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>407</width>
          <height>550</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="labelImagePreview">
           <property name="text">
            <string>abcd</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnTest1">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test1</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>881</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Metadata:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="txtMetadataDisplay">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Search History</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="history_view"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnTest2">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Test2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbUseThumbnail">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>310</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>use_thumbnail</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1512</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_config">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionImportFolder"/>
    <addaction name="actionImportEhViewerBackup"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menu_config"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImportFolder">
   <property name="text">
    <string>Import Folder</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionImportEhViewerBackup">
   <property name="text">
    <string>Import from EhViewer Backup</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tableSearchResult</tabstop>
  <tabstop>txtSearchBar</tabstop>
  <tabstop>btnTestListFullDb</tabstop>
  <tabstop>btnOpenSelectedFolder</tabstop>
  <tabstop>btnTestEhRequest</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
